src/enrichers/state-with-no-cookies-config.js

```diff
@@ -0,0 +1,11 @@
+export function enrich (state) {
+  if (state && state.gdprApplies != null) {
+    const gdprApplies = !!state.gdprApplies
+    return {
+      ...state,
```

Comment:
We should not build the final state here. Let's implement this enricher the same way others are implemented. The result then can be merged with the current state as we already do it with other enrichers.